pylint_odoo/test/main.py

```diff
@@ -9,6 +9,7 @@
 
 
 EXPECTED_ERRORS = 52
+EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
```

Comment:
Well, I prefer to let unfolded. As said, this allows to identify how many errors of each type you have introduced. If you have a global counter, you need to check each of them for a global number, and it more prone to put the given number by the test than the real expected one.